### ISSUE TITLE
InconsistentReferenceNameException arguments inconsistency

### ISSUE THREAD

**Issue by macieksmuga** [ga4gh/server#627]
`InconsistentReferenceNameException` is declared as follows in `exceptions.py`:
```
class InconsistentReferenceNameException(MalformedException):
    """
    A FASTA file has a reference name not equal to its file name.
    """
    def __init__(self, fileName):
        self.message = (
```

and invoked as follows in `references.py`:

```
            raise exceptions.InconsistentReferenceNameException(
                self._fastaFilePath, fastaFile.references[0])
```

Resulting in a `TypeError:  __init__() takes exactly 2 arguments (3 given)` when invoked.
(you can test its invocation by  giving the FASTA file a different name than the record within it - which is a problem in and of itself)

**Comment by jeromekelleher** [comment]
Oops, thanks @macieksmuga. We need some faultydata tests to exercise these cases.

**Comment by dcolligan** [comment]
Fixed in #629
